.circleci/config.yml

```diff
@@ -104,7 +107,8 @@ jobs:
       - restore_cache:
           key: repo-{{ .Environment.CIRCLE_SHA1 }}
 
-      - ruby/install-deps
+      - ruby/install-deps:
```

Comment:
This and similar lines below are grabbing a version-specific set of gems, correct?

Reply:
Yes. See https://circleci.com/developer/orbs/orb/circleci/ruby#commands-install-deps for what that is doing under the hood.